huaweicloud/config/endpoints.go

```diff
@@ -342,6 +342,13 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		WithOutProjectID: true,
 		Product:          "DNS",
 	},
+	"dnsv2.1": {
```

Comment:
```suggestion
	"dnsv21": {
```
don't forget to add it in multiCatalogKeys